pep-0590.rst

```diff
@@ -213,30 +280,25 @@ Removing any special cases and making all calls use the ``tp_call`` form was als
 However, unless a much more efficient way was found to create and destroy tuples, and to a lesser extent dictionaries,
 then it would be too slow.
 
+
 Acknowledgements
 ================
 
-Victor Stinner for developing the original "vector call" calling convention internally to CPython (where is it is called "fast call")
+Victor Stinner for developing the original "fastcall" calling convention internally to CPython (where is it is called "fast call")
```

Comment:
```suggestion
Victor Stinner for developing the original "vector call" calling convention internally to CPython (where is it is called "fast call")
```
Either leave this be, or remove the note in parentheses.

Reply:
Victor calls it "fastcall" not "vector call".